initcpio/hooks/bootstrap

```diff
@@ -29,12 +29,28 @@ process_mounts() {
       # has a valid target
       if [ ! -z "$target" ]; then
          dst="/new_root/$target"
-         mkdir -p ${dst} || true
+         mkdir -p "${dst}" || true
          echo "mounting $name on $target"
          mount "/dev/$name" "$dst"
       fi
    done
 }
+  
+# manually tested on export qsfs_qsfsnamewithoutdashesorcolons="/New Folder/targ=t"
+process_virtiofs_mounts() {
+env | while IFS= read -r entry; do
+      case $entry in
+      qsfs_*)
+            fs=${entry%%=*}
+            fs=${fs#qsfs_}
+            target=${entry#*=}
+            dst="/new_root/$target"
```

Comment:
as per the disk mount, add a check that the target is valid, and also an 'echo' for debugging.